volplugin/handlers.go

```diff
@@ -313,13 +210,14 @@ func (dc *DaemonConfig) mount(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := applyCGroupRateLimit(volConfig, mc); err != nil {
+	if err := applyCGroupRateLimit(volConfig.RuntimeOptions, mc); err != nil {
 		httpError(w, "Applying cgroups", err)
 		return
 	}
 
 	go dc.Client.HeartbeatMount(dc.Global.TTL, ut, dc.Client.AddStopChan(uc.Request.Name))
 
+	go dc.startRuntimePoll(volConfig.String(), mc)
```

Comment:
OH god, yes.

On 6 Apr 2016, at 8:56, Madhav Puri wrote:

>>  		httpError(w, "Applying cgroups", err)
>>  		return
>>  	}
>>
>>  	go dc.Client.HeartbeatMount(dc.Global.TTL, ut, 
>> dc.Client.AddStopChan(uc.Request.Name))
>>
>> +	go dc.startRuntimePoll(volConfig.String(), mc)
>
> In addition to handling mount, in case of volplugin restart we may 
> need to poll for already mounted volumes as well, right?
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/209/files/072305466f27065d492ea837ea4a57c7747e61fd#r58731697